src/pages/index.tsx

```diff
@@ -88,7 +88,7 @@ const modeDispatchPages: Record<AppMode, () => React.ReactElement> = {
               <Route path="/workspace/:uuid/hivechat/:chatId">
                 <HiveChatView />
               </Route>
-              <Route path="/hivechat/:uuid/build">
+              <Route path="/hivechat/:uuid/build/:chatId">
```

Comment:
This change doesn't make sense to me?

This is the current chat route:
/workspace/ck95pe04nncjnaefo08g/hivechat/cuvqckvpaij6rm5nbb90

Wouldn't the correct url be 
/workspace/ck95pe04nncjnaefo08g/hivechat/:chatID/build (which gives us the chat but on a different UI?)